tasks/igtf.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Set necessary facts
+  set_fact:
+    ca_branch: "unverified"
+    ca_server: admin-repo.egi.eu
+  when: ca_verification
+
+- name: Add apt key
+  apt_key:
+    url: "https://dist.eugridpma.info/distribution/igtf/current/GPG-KEY-EUGridPMA-RPM-3"
+    state: present
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Add IGTF repository for APT
+  apt_repository:
+    repo: deb http://{{ ca_server }}/sw/{{ ca_branch }}/cas/{{ ca_version }}/current egi-igtf core
+    state: present
+    filename: egi-igtf
+    update_cache: 'yes'
+  when: ansible_distribution == 'Ubuntu'
+
+- name: Ensure yum.repos.d directory is present
+  file:
+    path: /etc/yum.repos.d
+    owner: root
+    state: directory
+    group: root
+
+- name: Add IGTF repository for YUM
+  yum_repository:
+    name: EGI-trustanchors
+    description: EGI-trustanchors repo
+    baseurl: "http://{{ ca_server }}/sw/{{ ca_branch }}/cas/{{ ca_version }}/current/"
+    enabled: 'yes'
+    gpgcheck: 'yes'
+    gpgkey: http://repository.egi.eu/sw/production/cas/1/GPG-KEY-EUGridPMA-RPM-3
+  when: (ansible_os_family | lower) == 'redhat'
+
+- name: Ensure EGI Policy Core is present
+  package:
+    name: ca-policy-egi-core
+    state: present
+
+- name: Ensure WLCG CA package is present
+  package:
+    name: ca-policy-lcg
+    state: present
+  when: ca_verification
```

Comment:
Why is this `ca_verification` causing the inclusion of `ca-policy-lcg` ?

Reply:
the reason is that this role was first created for CMD verification means and here we need to validate all the packages, including the WLCG ones. Out of the scope of CMD verification I am not sure that the default behavior should be to install the WLCG package but just the EGI one.

Reply:
But without `ca_verification` IGTF repo and `ca-policy-egi-core` will still be installed, so it's a bit unlogical. Also I think it would probably be better to treat this role as first meant to be used to configure CMD, and have var names be related to this context and not that much linked to the verification process, it will ease use/reuse and have a broader scope than CMD verification.

Reply:
yes but my understanding is that `ca-policy-egi-core` should be the "reference" package for installing the trusted CAs for any EGI sw right?

With regards to the default behavior of the role, if we aim to have this role as the automated way to deploy EGI software repositories on (production) sites, the fact of having CMD verification as the default behavior is not the best practice I guess.